packages/twenty-front/src/modules/object-record/record-field/meta-types/input/components/ArrayFieldInput.tsx

```diff
@@ -0,0 +1,37 @@
+import { useArrayField } from '@/object-record/record-field/meta-types/hooks/useArrayField';
+import { ArrayFieldMenuItem } from '@/object-record/record-field/meta-types/input/components/ArrayFieldMenuItem';
+import { MultiItemFieldInput } from '@/object-record/record-field/meta-types/input/components/MultiItemFieldInput';
+import { useMemo } from 'react';
+
+type ArrayFieldInputProps = {
+  onCancel?: () => void;
+};
+
+export const ArrayFieldInput = ({ onCancel }: ArrayFieldInputProps) => {
+  const { persistArrayField, hotkeyScope, fieldValue } = useArrayField();
+
+  const arrayItems = useMemo<Array<string>>(
+    () => (Array.isArray(fieldValue) ? fieldValue : []),
+    [fieldValue],
+  );
+
+  return (
+    <MultiItemFieldInput
+      hotkeyScope={hotkeyScope}
+      newItemLabel="Add Item"
```

Comment:
+ Could we rename to `addNewItemLabel` ? 
+ @Bonapara is this the expected label:
<img width="198" alt="Capture d’écran 2024-09-06 à 11 26 09" src="https://github.com/user-attachments/assets/cb9cf787-fc38-451f-acda-c0f2fc01e792">

or shall we re-use the field's nameSingular (seems closer to this)? One drawback could be if the field name is long
![image](https://github.com/user-attachments/assets/3feece4e-4615-4f63-9a8f-994897a3bf3d)

Reply:
Hello @ijreilly 
we confirmed the label text with Bonapara [here](https://github.com/twentyhq/twenty/pull/6862#issuecomment-2326567032)